portals/devportal/src/main/webapp/source/src/app/components/Apis/Details/Overview.jsx

```diff
@@ -673,7 +673,7 @@ function Overview() {
                         </Typography>
                     </Box>
                     {(api.type === 'HTTP' || api.type === 'SOAPTOREST'
-                    || api.type === 'SOAP' || api.type === 'GRAPHQL') && (
+                    || api.type === 'SOAP' || api.type === 'GRAPHQL' || api.type === 'WS') && (
```

Comment:
Verify whether the `api.type` is always `WS` for any type of AsyncAPI i.e. WebSocket, WebHooks and SSE.

Reply:
There were also WEBSUB, SSE, and ASYNC Types. Added.